agnostic-angular/apps/examples/src/app/app.component.html

```diff
@@ -1094,17 +1154,97 @@ <h2>Dialog 2</h2>
                dialogRoot="#dialog-root"
                closeButtonPosition="last"
                [classNames]="{
-                container: 'my-dialog-container',
-                overlay: 'my-dialog-overlay',
-                document: 'my-dialog-content',
-                title: 'h4 mbe18',
-                closeButton: 'close-button-demo'
-               }"
+        container: 'my-dialog-container',
+        overlay: 'my-dialog-overlay',
+        document: 'my-dialog-content',
+        title: 'h4 mbe18',
+        closeButton: 'close-button-demo'
+      }"
                [closeButtonLastTemplate]="closeButtonLast2"
                [mainTemplate]="main2"
                [titleTemplate]="title2">
     </ag-dialog>
   </section>
+  <section>
+    <h2>Menus</h2>
+    <ag-menu class="my-menu"
+             menuTitle="Default Menu"
+             (triggerClick)="log('triggerFromConsumer', $event)">
+      <!-- Mark the Template for the menu components -->
+      <ng-template agMenuContent>
+        <button ag-menu-item
```

Comment:
I'm noticing that when I interact with this example the selected item is not showing the selected affordance (blue background) and I think it might be because we don't have the `[isSelected]="..."` part on this menu-item correct? I'll add this to the follow up ticket of todo items to fix.

Reply:
Yes, that's correct. I think there's a bigger conversation here about what makes sense for ag-ui. There's certainly a use case where you would want your menu items selection to persist, like the case of a theme picker. But I think there are also cases where you would _not_ want your menu to have selection state, like an "action menu". The truth of the matter is I was having a difficult time figuring out how to keep the state persisted, but it made me step back and consider that second use case. 

I kind of like having the ability to _not_ use the selection CSS in some cases. I kind of dislike that the ng API has diverged from the others.

Reply:
Oh ok, thanks for the clarification 👍  I like the idea of action menu use case -- maybe if we want we could add a little paragraph in the example or comment in the code. Later same for docs (which are evolving over time)